scipy/stats/stats.py

```diff
@@ -3543,19 +3547,22 @@ def count_rank_tie(ranks):
     x, y = x[perm], y[perm]
     x = np.r_[True, x[1:] != x[:-1]].cumsum(dtype=np.intp)
 
-    con, dis = _kendall_condis(x, y)  # concordant & discordant pairs
+    dis = _kendall_dis(x, y)  # discordant pairs
```

Comment:
Well, of course it can :). I mean, the previous code was computing just discordances. It's the new code that compute (uselessly) both.